packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -991,6 +991,9 @@ class _AppBarState extends State<AppBar> {
             TargetPlatform.iOS || TargetPlatform.macOS => null,
           },
           header: true,
+          // set default headingLevel: 1 for one h1 element
+          // per page to meet accessibility guidelines
+          headingLevel: 1,
```

Comment:
do we need this change to pass gar? I thought if we convert header to banner, It should have worked?

Reply:
I believe so, the [bug](b/338035526) I'm trying to fix references [GAR 1.6](go/GAR-web#overview-of-criterion-16) where it is required to have at least one h1 tag on each page. 

Let me know if this is a good way to enforce this rule.

Reply:
Attempted to make related engine changes for [issue](https://github.com/flutter/flutter/issues/152268) in [this PR](https://github.com/flutter/engine/pull/54224).

Reply:
This seems not addressed yet